dispersydatabase.py

```diff
@@ -84,255 +84,13 @@ def check_database(self, database_version):
             self.commit()
 
         else:
-            # upgrade an older version
+            # Check if the version is not higher than
+            if database_version > LATEST_VERSION:
+                raise RuntimeError(u"Your database version exceeds the latest version.")
```

Comment:
This might happen when we are shipping a pre-release experimental build and users decide to go back to the old (stable) version. Do we want to raise an Exception for that?